README.md

```diff
@@ -16,27 +10,39 @@ This tells you if your jobs are done. And stuff like that.
  - Gets input / output dataset names
  - Works with piping, other unix nice things
 
-## Installation
+Like badges? We have the best badges:
+
+[![GitHub Project](https://img.shields.io/badge/GitHub--blue?style=social&logo=GitHub)](https://github.com/dguest/pandamonium)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4019463.svg)](https://doi.org/10.5281/zenodo.4019463)
+
+[![PyPI version](https://badge.fury.io/py/pandamonium.svg)](https://badge.fury.io/py/pandamonium)
+[![Supported Python versions](https://img.shields.io/pypi/pyversions/pandamonium.svg)](https://pypi.org/project/pandamonium/)
+
+
+Installation
+------------
 
 ### Install from PyPI
 
 You can install [`pandamonium` from PyPI][pandamonium_PyPI] into any Python
-virtual environment by running
+environment by running
```

Comment:
oh noes. :disappointed: How will the children learn to Python if they keep on destroying their base environment? I gotta get my virtual environment evangelization in where I can my dude. #venv4greatergood

Reply:
They won't destroy their base environment because our code is environmentally conscious: as long as pip just installs this library and not 20 other things it should have a low impact and will play fine with everything else. (well, maybe this is part of my objection to including Panda libraries as a requirement, but this is another issue)

Reply:
Also, just to give my own anecdote here: I never use virtual environments on my laptop, and I get by fine. The reason is that I'm generally working on one or two things, and half of them are in containers anyway (which I admit is accomplishing the same thing). I certainly don't use a virtual environment _in_ a container.

For my part, I find it easier to periodically purge whatever I have installed in pip and start from scratch than to remember which environment I had set up 6 months ago for whatever project. Maybe I'm just lousy at multitasking, but my brain isn't able to keep track of more projects than my python installation is.

You're likely right that virtual environments are the right way to do things. But while I'm all for evangelizing, this implies that you _need_ a virtual environment to install this library, which isn't true. I think it's implied that a virtual environment will work if the base environment works, so I don't think we'll push off people who already like virtual environments.

Reply:
> But while I'm all for evangelizing, the previous wording implies that you need a virtual environment to install this library, which isn't true. 

I understand what you're saying that the target audience of physicists might not understand what virtual environments are. So I'd say sure go ahead and remove this.

---
(what follows here should probably be discussed elsewhere later)

While I think that this wording isn't the end of the world, your replies should probably be discussed in another chat or something as "why virtual environments should be the accepted way of working with Python" isn't necessarily what is being discussed on that line.

> It might sound nutty, but I find it easier to periodically purge whatever I have installed in pip and start from scratch than to remember which environment I had set up 6 months ago for whatever project. As long as the dependencies of these projects are clear, reinstallation sets me back about 3 minutes.

I would strongly recommend that you stop doing this. Especially because you're bound to have things start breaking the next time you update `pip` given the 2020 resolver change: https://discuss.python.org/t/announcement-pip-20-2-release/4863

> As long as the dependencies of these projects are clear, reinstallation sets me back about 3 minutes.

This is fundamentally not how dependencies work in Python. Python libraries are designed to work together with a wide but finite range of software. It is totally reasonable to have libraries required for one project that directly conflict with software that is required for another. This makes a global Python environment broken very quickly and very hard to fix at all. It is not very well communicated still, but I think the assumption of all library developers is that you are working in a Python virtual environment whenever you're installing their software.

I mentioned earlier that physicsts don't understand well what virtual environments are. I think that as a result, we should be pushing people to use them heavily, especially as there is way to much environment hacking that already goes on in physics that could be easily solved if people just used a virtual environment.

Reply:
Ok, I'm going to remove this. I'll add it back if you can convince me that I should never be using pip without a virtual environment. I'm excited for whatever horror stories we can dredge up.